containers/java-postgres/test-project/src/test/java/com/mycompany/app/AppTest.java

```diff
@@ -0,0 +1,44 @@
+/*--------------------------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+ *-------------------------------------------------------------------------------------------------------------*/
+
+package com.mycompany.app;
+
+import org.junit.Test;
+
+import junit.framework.AssertionFailedError;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.net.UnknownHostException;
+
+public class AppTest
+{
+    App newApp = new App();
+    public AppTest() {
+    }
+
+    @Test
+    public void testApp()
+    {
+        assertTrue( true );
+    }
+
+    @Test
+    public void testMore()
+    {
+        assertTrue( true );
+    }
+
+    @Test
+    public void testIP()
+    {
+        try {
+            assertTrue( App.pingAddress("postgresdb") );
```

Comment:
What is this test asserting? It seesms to establish that `postgresdb` is a routable name. It doesn't establish that there's a PostgresDb listening on that IP adress. If it's a valid test to assert that the containers are working as expected, then maybe change the error message below?

Reply:
Great point. I'll change the error message to be more prescriptive, I think that would cover it. We figured that the only real test we needed was to make sure that the container networking was set up correctly, and so this test establishes, as you said, that `postgresdb` can be reached.

Reply:
Changed the error messages and added more tests to be more specific about what we test.